reStream.sh

```diff
@@ -69,13 +69,6 @@ while [ $# -gt 0 ]; do
     esac
 done
 
-# technical parameters
-width=1408
```

Comment:
- width, height are switched on the RM2. 
- bytes per pixel is 1 on the RM2 
- i'm wondering if loop_wait might benefit from having a small sleep? Still yet to experiment

Reply:
I have experimented with sleeps, but it ended up not being beneficial I think. Anyway, we should speed up reading the framebuffer before changing `loop_wait`.